jogo/src/core/modal.ts

```diff
@@ -0,0 +1,16 @@
+import { ModalSkeleton } from "../entity/hud/modal_skeleton"
+import { Store } from "../infra/store"
+
+export class Modal {
+  private skeleton: ModalSkeleton
+  constructor(private templateName: string) {}
+
+  create(): void {
+    this.skeleton = new ModalSkeleton(new Store<string>(this.templateName))
+    this.skeleton.spawn()
+  }
+
+  destroy(): void {
+    this.skeleton.destroy()
+  }
+}
```

Comment:
eu esqueci de configurar um `.editorconfig` pra esse projeto forçando um newline no final da última linha, vc consegue configurar isso temporariamente só pra ele lembrar de colocar antes de salvar?

https://stackoverflow.com/questions/729692/why-should-text-files-end-with-a-newline

Reply:
adicionei o `.editorconfig` aqui ce9190932f682b9599aaca3eb6802a2d4f6dffca e modifiquei os 2 arquivos aqui ad4b6316ff06323de36204986dacc332acecd972